firmware/application/apps/acars_app.cpp

```diff
@@ -98,9 +98,11 @@ ACARSAppView::ACARSAppView(NavigationView& nav) {
 		logging = v;
 	};
 	
+	std::string folder = "LOGS";
+	make_new_directory(folder);
 	logger = std::make_unique<ACARSLogger>();
 	if (logger)
-		logger->append("acars.txt");
+		logger->append("LOGS/ACARS.TXT");
```

Comment:
It might make more sense to have the /LOGS/ as a constant so you can reference that everywhere instead of hard coding it. Also what's the reason for capitalising everything?

Reply:
Will do.
Capitalized: it's upper case everywhere and it was not consistent for these files. So I changed to all upper.

Reply:
Makes sense, I'm happy with that